apiary.apib

```diff
@@ -187,6 +187,21 @@ This is the Ribose API.
         * invisible
 * `members_count`: 10 (number) - how many members it contains
 
+### Space (object)
+
++ `id` (number) - space ID
```

Comment:
`id` should be of type `uuid`

Reply:
@ribose-jeffreylau FYI, most of the objects in `common_ds.apib` have ids of type id, not uuid. Is it correct, or another inconsistency?

Reply:
@skalee 

Here's the tricky bit --- some objects are `id` while others are `uuid`.

Currently, as a rule of thumb, one can expect Space and User object IDs are `uuid`.

As an aside, we expect to convert all `id` into `uuid` in the future.